tinygrad/runtime/ops_cuda.py

```diff
@@ -44,20 +44,21 @@ class device:
     get_device = lambda: context.device # pylint: disable=unnecessary-lambda # noqa: E731
   import pycuda.driver
   pycuda.driver.Context = context
-  RawCUDABuffer = RawMallocBuffer
 else:
   import pycuda.autoprimaryctx # pylint: disable=unused-import # noqa: F401
-  import pycuda.driver as cuda # type: ignore
+  import pycuda.driver as pycuda_driver
   class CUDAAllocator(LRUAllocator):
     def __init__(self): super().__init__(self._get_cur_free_space(None))
-    def _do_alloc(self, size, dtype, device, **kwargs): return cuda.mem_alloc(size * dtype.itemsize) # type: ignore
+    def _do_alloc(self, size, dtype, device, **kwargs): return pycuda_driver.mem_alloc(size * dtype.itemsize)
     def _cached_bufkey(self, size, dtype, device): return (device, size*dtype.itemsize) # Buffers of the same length could be reused, no matter what dtype.
-    def _get_cur_free_space(self, device): return cuda.mem_get_info()[0] # type: ignore
+    def _get_cur_free_space(self, device): return pycuda_driver.mem_get_info()[0]
   CUDAAlloc = CUDAAllocator()
-  class RawCUDABuffer(RawBufferCopyInOut): # type: ignore
+  class CUDAMemoryBuffer(RawBufferCopyInOut):
```

Comment:
I updated this with a slightly better name. We could call this something like PyCUDABuffer, but not sure if that's really a better name.